### ISSUE TITLE
Won't Start

### ISSUE THREAD

**Issue by gpresland** [node-opcua/opcua-commander#5]
After doing `npm install  opcua-commander -g` and trying to start the program, it errors with the following.

```
C:\Users\gpresland\AppData\Roaming\npm\node_modules\opcua-commander\node_modules\node-opcua\node_modules\better-assert\index.js:37
  throw err;
  ^
AssertionError: fs.existsSync(options.certificateFile)
    at OPCUAClientBase (C:\Users\gpresland\AppData\Roaming\npm\node_modules\opcua-commander\node_modules\node-opcua\lib\client\client_base.js:65:5)
    at new OPCUAClient (C:\Users\gpresland\AppData\Roaming\npm\node_modules\opcua-commander\node_modules\node-opcua\lib\client\opcua_client.js:67:21)
    at Object.<anonymous> (C:\Users\gpresland\AppData\Roaming\npm\node_modules\opcua-commander\index.js:91:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\gpresland\AppData\Roaming\npm\node_modules\opcua-commander\bin\opcua-commander:3:1)

```

**Comment by erossignon** [comment]
this is probably related to https://github.com/node-opcua/node-opcua/issues/212 . can you confirm ?

**Comment by gpresland** [comment]
The original stack trace posted was using node version 4.4.1 on Windows 10.
It was also tested on another machine with Ubuntu 14 with node version 6.2.2 and had the same error.

**Comment by erossignon** [comment]
I have update this package to use the latest node-opcua@0.0.55 version that should fix the node 6.2.2 issue.  Can you test it again ?

```
npm install opcua-commander -g 
```

**Comment by gpresland** [comment]
Tested it again and it's working now on 4.4.1 on Windows 10.
